core/src/Cuda/Kokkos_Cuda_Instance.hpp

```diff
@@ -130,6 +131,9 @@ class CudaInternal {
   mutable size_type* m_scratchUnified;
   mutable size_type* m_scratchFunctor;
   cudaStream_t m_stream;
+  cudaEvent_t m_last_event;
```

Comment:
I think this will have to be `mutable` too?

Reply:
No, we only access this member through a pointer to the internal class.